extensions/amazon-lambda/runtime/pom.xml

```diff
@@ -23,6 +23,13 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-lambda-java-events</artifactId>
+            <version>2.2.7</version>
```

Comment:
We shouldn't be overriding the version here, but instead change it here: https://github.com/quarkusio/quarkus/blob/master/bom/runtime/pom.xml#L139

Reply:
Sure. Updated.